azavea/cac-tripplanner-android
# Hide destination for events with multiple destinations

## Overview

Hide the destination for events that have multiple destinations.

Instead, show a message indicating that the event has multiple destinations, and link to the website details page.

## Demo

![image](https://user-images.githubusercontent.com/960264/69469669-15ae7e00-0d60-11ea-9ce6-d7e82cfc1468.png)

Destination not listed on event in list view:
![image](https://user-images.githubusercontent.com/960264/69469692-2bbc3e80-0d60-11ea-922e-25b0d9f23b5e.png)


## Notes

Since we are not fully implementing the new tours and multiple-destination events features in the Android app, instead we're simply hiding the destination from events that have more than one.

The old `destination` property is still present on the API response, and corresponds to the first destination when there are multiple.

## Testing

Testing this will require using the staging endpoint. Note that the one event currently on staging with multiple destinations is the one pictured above ("We Walk PHL in the Woodlands").

 - Modify the `WEBSERVICE_URL` in `DestinationWebservice.java` [here](https://github.com/azavea/cac-tripplanner-android/blob/develop/app/src/main/java/org/gophillygo/app/data/DestinationWebservice.java#L22) to point to https://staging.gophillygo.org/.
 - Also modify the `website_host` string added to `strings.xml` here to point to staging (used to link out to the event details page).
 - Install and run app
 - Go to events list
 - Destination should not be listed for event with multiple destinations
 - Tap to go to details for event with multiple destinations
 - Should see extra section noting multiple destinations, with a link
 - Link should go to event details page on the site
 - Other events (with one or no destination) should not display the new section


Closes #173.
